src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -40,22 +41,33 @@ interface IAccessibleBufferQuickPickItem extends IQuickPickItem {
 }
 
 export class AccessibleBufferWidget extends DisposableStore {
+
 	private _accessibleBuffer: HTMLElement;
 	private _editorWidget: CodeEditorWidget;
 	private _editorContainer: HTMLElement;
 	private _xtermElement: HTMLElement;
+
 	private readonly _focusedContextKey: IContextKey<boolean>;
 	private readonly _focusTracker: dom.IFocusTracker;
 
+	private _lastMarker: IMarker | undefined;
+	private _lastLinesInViewport: number = 0;
+	private _lines: string[] = [];
+	get lines(): string[] { return this._lines; }
```

Comment:
Instead of making it public, make it protected and then create a TestAccessibleBufferWidget that exposes the protected field, similar to this: https://github.com/weartist/vscode/blob/e68e0ba27220dcf3aa579b08c5e6132b6ae06712/src/vs/workbench/contrib/terminal/test/browser/terminalTypeahead.test.ts#L390